### PR TITLE
[Project Addition]: Almond Varieties Image Classification

### PR DESCRIPTION
**Pull request by jeet-Abhi123**
# Pull Request for DL-Simplified 💡

## Issue Title : Almond Varieties Image Classification 

- **Info about the related issue (Aim of the project)** :  We aim to demonstrate the practical application of deep learning models in agricultural and industrial contexts, showcasing how technology can enhance productivity and accuracy in classification tasks.
- **Name:** Abhijeet Kaithwas
- **GitHub ID:**  [GitHub ID](https://github.com/jeet-Abhi123) 
- **Email ID:** abhijeet.kaithwas13@gmail.com 
- **Idenitfy yourself: (Mention in which program you are contributing in. )**  GSSOC'24 

Closes: #740 

### Describe the add-ons or changes you've made 📃
Give a clear description of what have you added or modifications made

I have implemented the InceptionV3, VGG19 & ResNet50 models with proper image classification techniques.

## Checklist: ☑️

- [x] My code follows the guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly wherever it was hard to understand.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [x] I have added things that prove my fix is effective or that my feature works.
- [x] Any dependent changes have been merged and published in downstream modules.

### CODE REVIEW

File: Almond Varieties Image Classification/README.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+**VGG19 Performance**
+![VGG19_results](https://github.com/jeet-Abhi123/Road-Safety-Data-Analysis-Power-BI-/assets/143840497/e648c009-1a20-4662-9f5a-72d39a88be3c)
+
+**ResNet50 Performance**
+![ResNet_results](https://github.com/jeet-Abhi123/Road-Safety-Data-Analysis-Power-BI-/assets/143840497/709f2625-7d23-499b-9af9-71014c1c063f)
+
+### 📢 Conclusion
+This project successfully demonstrates the use of advanced deep learning models to classify different varieties of almonds. By leveraging InceptionV3, VGG19, and ResNet50, the project highlights the effectiveness and performance of these architectures in image classification tasks.
+
+###Accuracy Results
```
Comment (abhisheks008) [inline Almond Varieties Image Classification/README.md:59]: Typing mistake.

**Review by abhisheks008 — CHANGES_REQUESTED** [review]
Need to do some EDA before directly implementing the models. Add the EDA results inside the Images folder.

**Comment by jeet-Abhi123** [comment]
@abhisheks008  
Made the necessary changes.

**Review by abhisheks008 — APPROVED** [review]
Approved @jeet-Abhi123

**Pull request merged**